controller/node_controller.go

```diff
@@ -381,6 +381,7 @@ func (nc *NodeController) syncNode(key string) (err error) {
 		condition.Reason = string(types.NodeConditionReasonManagerPodMissing)
 		condition.Message = fmt.Sprintf("manager pod missing: node %v has no manager pod running on it", node.Name)
 		node.Status.Conditions[types.NodeConditionTypeReady] = condition
+		node.Spec.AllowScheduling = false
```

Comment:
User can still set scheduling to false when the node is offline. So we shouldn't need to do this.